Decompress Run-Length Encoded List.py

```diff
@@ -0,0 +1,9 @@
+class Solution:
+    def decompressRLElist(self, nums: List[int]) -> List[int]:
+        lst = []
+        for i in range(len(nums)-len(nums)//2):
```

Comment:
why `len(nums) - len(nums)//2` rather than just `len(nums)//2`, wouldn't it be the same?